googledrive-repository/pom.xml

```diff
@@ -13,8 +13,8 @@
 
     <properties>
         <org.springframework.social-google-version>1.0.0.RELEASE</org.springframework.social-google-version>
-        <alfresco-community.version>6.38</alfresco-community.version>
-        <alfresco-enterprise.version>6.38</alfresco-enterprise.version>
+        <alfresco-community.version>6.44</alfresco-community.version>
```

Comment:
Just for future reference, how did we decide these versions ? Was it based on following tags / links ... ?

Com: https://github.com/Alfresco/acs-community-packaging/blob/acs-community-packaging-6.0.5-ea/pom.xml#L67
Ent: https://github.com/Alfresco/acs-packaging/blob/acs-packaging-6.0.0-RC5/pom.xml#L39